docs/source/grid.rst

```diff
@@ -0,0 +1,81 @@
+LUCID Grid Configuration
+========================
+
+``LUCID``'s main grid area is an auto-generated series of buttons and indicators,
```

Comment:
```suggestion
``lucid``'s main grid area is an auto-generated series of buttons and indicators,
```

Should we keep this consistent throughout the docs?

Reply:
I'm trying to use `LUCID` when talking about the app and `lucid` when talking about the cli invocation, but:

- I may not have done this correctly in general
- This might not be the right approach

I'm tempted to turn it all into `lucid` in general and `Lucid` in titles

Reply:
A lot of this stems from the readme and docs previously using `LUCID` exclusively despite the lowercase cli entrypoint + my dislike of all caps or special caps names in general

Reply:
I feel like my opinion on the matter changes daily.  I think it's fine as is, it just stuck out as the only fully-capitalized instance on the page